circuits/cpp/barretenberg/cpp/bootstrap.sh

```diff
@@ -35,7 +35,7 @@ if [ "$OS" == "macos" ]; then
   PRESET=default
 else
   if [ "$(which clang++-15)" != "" ]; then
```

Comment:
Shouldn't this if condition be changed too?

Reply:
ah yeah - if there's automerge and you have feedback like this you can not approve